### PR TITLE
Skip CI on backport branches

### PR DESCRIPTION
**Pull request by nabobalis**
Should skip backports branches on all CI.

### CODE REVIEW

**Comment by nabobalis** [comment]
CircleCi skips the branch even if a PR is opened.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+no-backports: &no-backports
+  name: Skip any branches called backport* but not PRs
+  command: |
+    if [[ -z "${CIRCLE_PR_NUMBER}" && "${CIRCLE_BRANCH}" = *"backport"* ]]; then
```
Comment (nabobalis) [inline .circleci/config.yml]: My logic is either broken or `CIRCLE_PR_NUMBER` is blank for a PR unless its a forked PR.

**Comment by nabobalis** [comment]
This now seems to work.

**Comment by wtbarnes** [comment]
Why do the py37-online and figure tests fail?

**Comment by nabobalis** [comment]
I will restart them. I created a second branch with the same commits and the CI look for commit numbers so when I deleted the second branch it errored.

**Comment by nabobalis** [comment]
You can  check the status of https://github.com/sunpy/sunpy/branches/active for the two branches.

**Pull request merged**